torchdynamo/optimizations/backends.py

```diff
@@ -102,6 +102,12 @@ def nvfuser_ofi(subgraph):
         return reload_jit_model_ofi(subgraph)
 
 
+@create_backend
+def onednn_graph(subgraph):
```

Comment:
```suggestion
def onednn(subgraph):
```
Using the string `"onednn"` seems nicer than `"onednn_graph"`

Reply:
Agree, that's nice without `_`. But `onednn_graph` is a bit different than onednn, thus using that might be miss leading. Let me think if there's a better naming.

Reply:
good suggest. I'll confirm the naming with our team. Thanks.

Reply:
The final call is up to you, happy to merge this name too.

Reply:
@jansel I updated the name. Thanks for suggesting.